docs/extensions/life-cycle-events/index.mdx

```diff
@@ -235,22 +235,22 @@ function waitForRequest(method: string, url: string) {
   let requestId = ''
 
   return new Promise((resolve, reject) => {
-    server.events.on('request:start', (req) => {
+    server.on('request:start', (req) => {
       const matchesMethod = req.method.toLowerCase() === method.toLowerCase()
-      const matchesUrl = matchRequestUrl(url, req.url)
+      const matchesUrl = matchRequestUrl(req.url, url);
```

Comment:
This is a good one. According to the [call signature](https://github.com/mswjs/msw/blob/5b0336c8970daa9db69876c20be7d6ffa8de5e60/src/utils/matching/matchRequestUrl.ts#L38), the actual URL must come first, followed by the path (expected URL).